test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/NoDepsAppTests.cs

```diff
@@ -79,6 +80,11 @@ public void RestartProcessThatTerminatesAfterFileChange()
                 // Then wait for it to restart when we change a file
                 using (var wait = new WaitForFileToChange(scenario.StartedFile))
                 {
+                    // On Unix the file write time is in 1s increments;
+                    // if we don't wait, there's a chance that the polling
+                    // watcher will not detect the change
+                    Thread.Sleep(1000);
```

Comment:
We could probably poll but it would be extra logic for such a small gain. To poll you'd have to go to the disk, get the file, get the file's latest write time, see if the current time is at least 1 second later than that. If not, wait a bit and either do that again or wait 1 second anyway. That's a lot of logic for the hope that you can save at most 1 second in a test run that takes 3-4 minutes